metrics/cli/docs/diagrams/deploy_command_sequence_diagram.puml

```diff
@@ -71,19 +68,26 @@ Bob -> FirebaseService : enters any key
 FirebaseService --> Deployer -- : firestore has been deployed
 
 
-Deployer -> FlutterService ++ : build()
-FlutterService -> Deployer : build finished
-FlutterService --
+Deployer -> FlutterService ++ : build(appPath)
+FlutterService -> Deployer -- : build finished
+
+Deployer -> SentryService ++ : login()
+SentryService --> Bob : starts interactive login
+Bob -> SentryService : signs in
+SentryService --> Deployer -- : sign in finished
+
+Deployer -> SentryService ++ : createRelease(appPath)
+SentryService -> FileHelper ++ : updateFile(config, options)
+FileHelper -> SentryService -- : file has been updated
+SentryService -> Deployer -- : release created
```

Comment:
It seems like the `SentryService` is interacting with the user during release creation. Should we display this interaction on this diagram?